src/initial-submission/components/DetailsForm.tsx

```diff
@@ -30,14 +30,39 @@ const defaultManuscriptDetails = (values: ManuscriptDetails): ManuscriptDetails
 };
 
 const DetailsForm = ({ initialValues, ButtonComponent }: Props): JSX.Element => {
+    const { t } = useTranslation('wizard-form');
+    // TODO: hook this up to useForm. Bugs in this page preventing this currently
```

Comment:
I'm guessing this is going to get followed up on soon, and not left here by accident?

Reply:
Have added comment to PR, we'll hook this up as part of fixing the auto-save issues